sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -398,26 +472,6 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
-  test("create hive view for json table") {
```

Comment:
The comment of @jiangxb1987 is out of dated. After leaving this comment, he moved the test cases to `SQLViewSuite` based on my comments.